.travis.yml

```diff
@@ -1,15 +1,36 @@
 language: php
 
+sudo: false
+
+cache:
+    directories:
+        - $HOME/.composer/cache
+
 php:
     - 5.3
     - 5.4
     - 5.5
     - 5.6
+    - 7.0
     - hhvm
 
-before_script:
-    - "composer require --dev phpspec/phpspec:~2.0.0-RC4 --no-update"
-    - "composer install --no-progress --no-plugins"
+env:
+    global:
+        - TEST_COMMAND="composer test"
+
+matrix:
+    allow_failures:
+        - php: hhvm
```

Comment:
Yeah, I usually add hhvm to fail by default to see if it actually works or not. Then remove from failure if works.